app/shared/src/buttons/ButtonWrapper.js

```diff
@@ -0,0 +1,45 @@
+// @flow strict
+
+import * as React from 'react';
+
+import Color from '../Color';
+import Touchable from '../Touchable';
+import StyleSheet from '../PlatformStyleSheet';
+import type { StylePropType } from '../../index';
+
+type Props = {|
+  children: React.Node,
+  onPress?: () => void,
+  style?: StylePropType,
+|};
+
+export default function ButtonWrapper(props: Props) {
```

Comment:
Not sure about the naming, it sounds like I am wrapping a button, but it is really a button

Reply:
Suggestions?

Reply:
Naming is hard...

Reply:
No, it's not that complicated... 🙂 Can I rename this to the `Button` and create `TextButton` instead of our Button.js?

Reply:
Not a deal-break though, I am fine merging it with this name

Reply:
> Can I rename this to the Button and create TextButton instead of our Button.js